.env

```diff
@@ -3,15 +3,14 @@
 # Used to configure host-ports>
 DQANALYZER_PORT="9090"
 DQAPI_PORT="8099"
+DQEDIT_PORT="8070"
 DQTRANSLATOR_PORT="8080"
 DQEXEC_PORT="8090"
-MONGO_PORT="27017"
 MONGO_MANAGEMENT_PORT="8081"
 INFLUX_PORT="8086"
 GRAFANA_PORT="3030"
 SAMPLE_APPLICATION_PORT="9000"
-RABBIT_PORT="15672"
-RABBIT_MANAGEMENT_PORT="5672"
+RABBIT_MANAGEMENT_PORT="15672"
```

Comment:
Seems like there was a mistake with the rabbit-port.
Why did you delete the Mongo- and Rabbit-Ports?

Reply:
Rabbit and Mongo don't need to be exposed, since they don't need to be accessed from outside of docker (just like dqexec, dqtranslator...). I only want to expose the management ui - for that thats the right port

Reply:
Makes sense!:)